### PR TITLE
add support for plugin cases

### PR DESCRIPTION
**Pull request by yapei**
Ready for review

### CODE REVIEW

**Comment by yapei** [comment]
@liangxia Could you help review?

File: features/step_definitions/web.rb [hunk 1]
```diff
@@ -315,4 +315,18 @@
...
+end
+
+Given /^admin enable demo dynamic plugin successfully$/ do
+  ensure_admin_tagged
+  full_version, major, minor = env.get_version(user: user)
+  plugin_image_tag = full_version.gsub('.','')
+  demo_plugin_image = 'quay.io/openshifttest/console-demo-plugin:' + plugin_image_tag
+  step %Q|I obtain test data file "templates/oc-manifest-demo-plugin.yaml"|
+  step %Q/I replace lines in "oc-manifest-demo-plugin.yaml":/, table(%{
+    | image: quay.io/vszocs/console-demo-plugin | image: #{demo_plugin_image} |
```
Comment (liangxia) [inline features/step_definitions/web.rb:327]: Perhaps put `quay.io/openshifttest/console-demo-plugin` in the yaml file ?

Comment (yapei) [reply features/step_definitions/web.rb]: updated, PTAL

**Comment by yapei** [comment]
@liangxia  updated, PTAL

File: features/step_definitions/web.rb [hunk 2]
```diff
@@ -315,4 +315,17 @@
...
\ No newline at end of file
+end
+
+Given /^admin enable demo dynamic plugin successfully$/ do
+  ensure_admin_tagged
+  full_version, major, minor = env.get_version(user: user)
+  plugin_image_tag = full_version.gsub('.','')
+  step %Q|I obtain test data file "templates/oc-manifest-demo-plugin.yaml"|
+  step %Q/I replace lines in "oc-manifest-demo-plugin.yaml":/, table(%{
+    | :latest | : #{plugin_image_tag}} |
```
Comment (liangxia) [inline features/step_definitions/web.rb:326]: An extra white space between `: #` ?

Comment (yapei) [reply features/step_definitions/web.rb:326]: you are right, sorry typo

Comment (yapei) [reply features/step_definitions/web.rb]: updated, thanks @liangxia for your checking

**Pull request merged**